joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/AstCreator.scala

```diff
@@ -26,14 +27,44 @@ class AstCreator(val config: Config, val parserResult: ParserResult)
 
   override def createAst(): DiffGraphBuilder = {
 
-    val fileNode = NewFile().name(parserResult.filename).order(1)
-
-    val astForDeclaration = parserResult.json(ParserKeys.Decls).arr.map(item => astForNode(item))
-    val ast               = Ast(fileNode).withChildren(astForDeclaration)
+//    val fileNode = NewFile().name(parserResult.filename).order(1)
+    val rootNode = createParserNodeInfo(parserResult.json)
+    val ast      = astForTranslationUnit(rootNode)
     Ast.storeInDiffGraph(ast, diffGraph)
     diffGraph
   }
 
+  private def astForTranslationUnit(rootNode: ParserNodeInfo): Ast = {
+    val fullQualifiedPackage =
+      GoMod.getNameSpace(parserResult.filename, parserResult.json(ParserKeys.Name)(ParserKeys.Name).str)
+    val namespaceBlock = NewNamespaceBlock()
+      .name(fullQualifiedPackage)
+      .fullName(s"$relPathFileName:${fullQualifiedPackage}")
```

Comment:
```suggestion
      .fullName(s"$relPathFileName:$fullQualifiedPackage")
```

Reply:
Also rename `fullQualifiedPackage` to `fullyQualifiedPackage`

Reply:
done